tests/modelica/test_modelica_runner.py

```diff
@@ -179,4 +169,61 @@ def test_simulate_msl_with_starttimes_in_docker(self):
         self.assertTrue(os.path.exists(os.path.join(results_path, 'stdout.log')))
         self.assertTrue(os.path.exists(os.path.join(results_path, f'{model_name}_res.mat')))
 
-        
\ No newline at end of file
+    @pytest.mark.simulation
+    def test_simulate_mbl_pid_in_docker(self):
+        model_name = 'Buildings.Controls.OBC.CDL.Continuous.Validation.PID'
+        
+        mr = ModelicaRunner()
+        success, _ = mr.run_in_docker(
+            'compile_and_run', model_name, run_path=self.mbl_run_path, project_in_library=True
+        )
+        self.assertTrue(success)
+
+    @pytest.mark.dymola
+    def test_simulate_msl_in_dymola(self):
+        model_name = 'Modelica.Blocks.Examples.PID_Controller'
+        results_path = Path(self.msl_run_path) / f"{inspect.currentframe().f_code.co_name}_results"
+        if results_path.exists():
+            shutil.rmtree(results_path, ignore_errors=True)
+        results_path.mkdir(parents=True)
+                
+        # shutil.rmtree(results_path, ignore_errors=True)
+
+        mr = ModelicaRunner()
+        success, _ = mr.run_in_dymola(
+            'simulate', model_name, run_path=results_path, file_to_load=None
+        )
+
+        self.assertTrue(success)
+        # self.assertTrue(os.path.exists(os.path.join(results_path, 'stdout.log')))
+        # self.assertTrue(os.path.exists(os.path.join(results_path, f'{model_name}.fmu')))
+
+    @pytest.mark.dymola
+    def test_simulate_mbl_pid_in_dymola(self):
+        results_path = Path(self.mbl_run_path) / f"{inspect.currentframe().f_code.co_name}_results"
+        if results_path.exists():
+            shutil.rmtree(results_path, ignore_errors=True)
+        results_path.mkdir(parents=True)
+
+        model_name = 'Buildings.Controls.OBC.CDL.Continuous.Validation.PID'
+        
+        mr = ModelicaRunner()
+        success, _ = mr.run_in_dymola(
+            'simulate', model_name, run_path=results_path, file_to_load=None  #, debug=True
+        )
+        self.assertTrue(success)
+
+    @pytest.mark.dymola
+    def test_compile_mbl_pid_in_dymola(self):
+        results_path = Path(self.mbl_run_path) / f"{inspect.currentframe().f_code.co_name}_results"
+        if results_path.exists():
+            shutil.rmtree(results_path, ignore_errors=True)
+        results_path.mkdir(parents=True)
+
+        model_name = 'Buildings.Controls.OBC.CDL.Continuous.Validation.PID'
+        
+        mr = ModelicaRunner()
+        success, _ = mr.run_in_dymola(
+            'compile', model_name, run_path=results_path, file_to_load=None, debug=True
+        )
+        self.assertTrue(success)
```

Comment:
```suggestion
        self.assertTrue(success)
        
```

Add a newline to keep IDE from whining 😆